packages/design-system-scripts/gulp/sass.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const autoprefixer = require('autoprefixer');
+const getDocsDistPath = require('./common/getDocsDistPath');
+const changed = require('gulp-changed');
+const count = require('gulp-count');
+const cssnano = require('cssnano');
+const gulp = require('gulp');
+const gulpIf = require('gulp-if');
+const path = require('path');
+const postcss = require('gulp-postcss');
+const postcssImport = require('postcss-import');
+const postcssInliner = require('postcss-image-inliner');
```

Comment:
Do we need this in the future? I thought we decided to stop inlining images?

Reply:
Yep, good catch

Reply:
Yesterday I was trying to find some info online about inlining vs minification. Do you have any context on that, aside from simplifying our build scripts and assets? I was thinking one other advantage might be that the CSS bundle will be slightly smaller (images can be loaded after).